app/javascript/components/pages/Home.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import Home from "./Home";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("When Home renders", () => {
+  const patientsArray = [];
+  it("displays a header", () => {
+    const home = shallow(<Home patientsArray={patientsArray} />);
+    const homeHeading = home.find("h1").text();
+    expect(homeHeading).toEqual("Patients");
+  });
+});
+
+// describe("When home renders", () => {
```

Comment:
You should not leave commented out code in your PRs.

Reply:
deleted